src/app/views/authentication/Authentication.tsx

```diff
@@ -1,182 +1,134 @@
-import hello from 'hellojs';
 import React, { Component } from 'react';
-
 import { connect } from 'react-redux';
 import { bindActionCreators, Dispatch } from 'redux';
-
 import { IAuthenticationProps, IAuthenticationState } from '../../../types/authentication';
 import * as authActionCreators from '../../services/actions/auth-action-creators';
 import * as queryActionCreators from '../../services/actions/query-action-creators';
-import { ADMIN_AUTH_URL, AUTH_URL, DEFAULT_USER_SCOPES , TOKEN_URL,
-  USER_INFO_URL, USER_PICTURE_URL } from '../../services/constants';
 import SubmitButton from '../common/submit-button/SubmitButton';
 import './authentication.scss';
+import { getAccessToken, getImageUrl, getUserInfo, logOut } from './AuthService';
 import { Profile } from './profile/Profile';
 
 export class Authentication extends Component<IAuthenticationProps,  IAuthenticationState> {
-  public state = {
-    authenticated: {
-      status: false,
-      user: {
-        displayName: '',
-        emailAddress: '',
-        profileImageUrl: '',
+  constructor(props: IAuthenticationProps) {
+    super(props);
+    this.state = {
+      authenticatedUser: {
+        status: false,
+        user: {
+          displayName: '',
+          emailAddress: '',
+          profileImageUrl: '',
+        },
+        token: '',
       },
-      token: '',
-    },
-    loading: false,
-  };
+      loading: false,
+    };
+  }
 
   public componentDidMount = () => {
-    const authenticated = localStorage.getItem('authenticated');
-    if (authenticated && this.props.actions && JSON.parse(authenticated).status) {
-      this.props.actions.authenticateUser(JSON.parse(authenticated));
+    const authenticatedUser = localStorage.getItem('authenticatedUser');
+    const authUser = (authenticatedUser) ? JSON.parse(authenticatedUser) : null;
+    if (authenticatedUser && this.props.actions && authUser.status) {
```

Comment:
Add a new line between the variable declaration and the if statements

Reply:
lines added. However, aesthetics are enforced by the linter. We should consider modifying the linter as it disapproves the lines and forces me to add two lines, and then remove one for the linter